tools/alphafold/alphafold.xml

```diff
@@ -37,10 +37,10 @@
 
         ## run alphafold  -------------------------
         ln -s /app/alphafold/alphafold alphafold &&
-  	    python /app/alphafold/run_alphafold.py
-  	    --fasta_paths alphafold.fasta
+  	python /app/alphafold/run_alphafold.py
+  	--fasta_paths alphafold.fasta
         --output_dir output
-        --data_dir /data                        ## location of the alphafold databases on pulsar node
+        --data_dir /data                   ## location of the alphafold databases on pulsar node --> could this maybe a env var? $ALPHAFOLD_DB --> \${ALPHAFOLD_DB:-/data}

```

Comment:
We could make those directories configurable via ENV vars if we like.

Reply:
I really like this idea.  For other admins creating their own alphafold job destinations this would be more seamless. Just need to mention that $ALPHAFOLD_DB needs to be configured.